src/functions/index.js

```diff
@@ -24,4 +24,18 @@ export function shuffle(array) {
   }
   
   return tempArray;
-}
+};
+
+export function sliceDeckInThree(deck) {
+  const deckToSplice = deck;
+  
+  const initialStack = deckToSplice.splice(0, 7); 
+  const middleStack = deckToSplice.splice(0, 7); 
+  const lastStack = deckToSplice.splice(0, 7); 
```

Comment:
Aqui ainda tem o probleminha do "Magic Number" com esse `7`.
Eu sei que o deck tem 21 cartas, e que a função vai dividir esse deck em 3 partes iguais, logo serão 7 cartas por `satck`. 
Mas sem esse context todo, eu não saberia o que significa o número `7`. Então vale a pena "dar nome aos bois", talvez algo como:
```suggestion
const QUANTITY_OF_CARDS_PER_STACK = 7;

export function sliceDeckInThree(deck) {
  const deckToSplice = deck;
  
  const initialStack = deckToSplice.splice(0, QUANTITY_OF_CARDS_PER_STACK); 
  const middleStack = deckToSplice.splice(0, QUANTITY_OF_CARDS_PER_STACK); 
  const lastStack = deckToSplice.splice(0, QUANTITY_OF_CARDS_PER_STACK); 
```